data-mapping/ci.sh

```diff
@@ -0,0 +1,3 @@
+sh prepare_data.sh
```

Comment:
All of these should have shell scripts should have `set -e`

Reply:
Done!